sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1136,6 +1150,21 @@ object Radar {
                 if (status == RadarStatus.SUCCESS) {
                     RadarSettings.setTripOptions(context, options)
 
+                    // store previous tracking options for post-trip
+                    // if tracking was false, previousTrackingOptions will be null
+                    val isTracking = Radar.isTracking()
+                    if (isTracking) {
+                        val previousTrackingOptions = Radar.getTrackingOptions()
```

Comment:
One thing about [Radar.getTrackingOptions()](https://github.com/radarlabs/radar-sdk-android/blob/develop/sdk/src/main/java/io/radar/sdk/Radar.kt#L1030) is that it pulls from RTO and then falls back to client TO if that's not set.  You probably want to be reading and writing to the client TO directly from RadarSettings.

Have a look at https://github.com/radarlabs/radar-sdk-android/blob/develop/sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt#L236, just for more context, this is where the RTO get used.

Reply:
cc @bschoeneweis @corypisano @palmakkar